lib/auth/accountrecovery_test.go

```diff
@@ -1334,3 +1378,25 @@ func getMockedU2FAndRegisterRes(authSrv *Server, tokenID string) (*proto.U2FRegi
 		ClientData:       u2fRegResp.ClientData,
 	}, u2fKey, nil
 }
+
+func getMockedU2FAndRegisterRes(authSrv *Server, tokenID string) (*wantypes.CredentialCreationResponse, *mocku2f.Key, error) {
```

Comment:
```suggestion
func getMockedWebauthnAndRegisterRes(authSrv *Server, tokenID string) (*wantypes.CredentialCreationResponse, *mocku2f.Key, error) {
```

Btw, any change we can replace this with a TestDevice / NewTestDeviceFromChallenge?